index.html

```diff
@@ -284,6 +297,22 @@ <h4>View only</h4>
 					size="medium"
 				></ts-file-card>
 			</article>
+
+			<article>
+				<h2>ts-help-text</h2>
+				<ts-help-text
+					messages='["Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias, at atque deserunt, dignissimos dolor doloribus eaque eius eos eum excepturi fuga, hic incidunt itaque molestias necessitatibus pariatur rem repellat soluta", "message example 1", "message example 2"]'
+					title="Title"
+					size="small"
+				></ts-help-text>
+				<ts-help-text
+					messages='["Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias, at atque deserunt, dignissimos dolor doloribus eaque eius eos eum excepturi fuga, hic incidunt itaque molestias necessitatibus pariatur rem repellat soluta", "message example 1", "message example 2"]'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 282 characters long line (which is greater than 160 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AMaxLineLengthCheck)